main.js

```diff
@@ -29,9 +29,9 @@ function main(args) {
         .usage('[options]')
         .option('-l, --listen <listen>', 'Host and port on which server listens (also node\'s identity in cluster)')
         .option('-h, --hosts <hosts>', 'Seed file of list of hosts to join')
-        .option('--suspect [suspect]', 'Suspect period in seconds')
-        .option('--faulty [faulty]', 'Faulty period in seconds')
-        .option('--tombstone [tombstone]', 'Tombstone period in seconds')
+        .option('--suspect-period <suspectPeriod>', 'Suspect period in ms', parseInt10, 5000)
```

Comment:
parseInt10 is not a global function in node (but parseInt is exactly what you want).